posts/data+ai-summit-top-announcements-and-recap.md

```diff
@@ -0,0 +1,68 @@
+---
+authors:
+- Theo Lebrun
+tags:
+- Data
+- Databricks
+- Apache Spark
+date: 2022-07-01T17:27:07.000Z
+title: "Data+AI Summit 2022 - Top Announcements and Recap"
+image: 
+---
+
+[Data+AI Summit 2022](https://databricks.com/dataaisummit/) is the world’s largest gathering of the data and analytics community and it took place in San Francisco or virtually during 4 days! With a very large number of speakers that are top expert across the data and AI ecosystem, the conference was a huge success in term of attendees (~5k in person and ~50K virtually). With a packed agenda mixing Keynotes, 30mins talks or 2hours industry forums, the conference was full of exciting announcements and very interesting sessions. A large number of trainings and workshops were also available for everyone from data engineers to data scientists to business leaders in order to improve any data skills.
+
+![](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/07/data-ai-summit.png)
+
+# Top Announcements
+
+## Delta Lake 2.0
+
+The Delta format is based on Apache Parquet and is the standard storage layer for your Databricks Lakehouse. It provides a lot of features like ACID transactions, audit history, time travel, etc (check this [article](https://databricks.com/blog/2022/06/30/open-sourcing-all-of-delta-lake.html) for more details).
+
+By open-sourcing it, Databricks will let you produce Delta files that other applications will be able to read and use! That allows you to take advantage of all the features previously quoted while enjoying up to 4.3x faster processing compared to other storage layers.
+
+## Project Lightspeed
+
+With more and more real-time data pipelines and infrastructures, streaming data is becoming technically challenging. And, it has needs far different from and more complicated to meet than those of event-driven applications and batch processing.
+
+Databricks is releasing Project Lightspeed, a framework which advances Spark Structured Streaming into the real-time era as more and more new use cases and workloads migrate into streaming ([more details here](https://databricks.com/blog/2022/06/28/project-lightspeed-faster-and-simpler-stream-processing-with-apache-spark.html)).
+
+## Spark Connect
+
+Spark Connect is a simple client-server protocol that will let you run Spark anywhere! The demo on stage was done using an iPad but the possibilities are infinite (phones, embedded systems, etc).
+
+After configuring your Spark client, it will generate a query plan of your Spark code and send it to your Spark cluster so it can do the heavy work for you. You will be able to run Spark directly on your laptop without having to have a local cluster that uses all your resources.
```

Comment:
```suggestion
After configuring your Spark client, it will generate a query plan of your Spark code and send it to your Spark cluster so it can do the heavy lifting for you. You will be able to run Spark directly on your machine without having to have a local cluster using all your resources.
```